docs/markdown/android/checkbox.md

```diff
@@ -0,0 +1,135 @@
+    - When a selection takes immediate effect, especially on mobile. Use [Switch](https://gestalt.pinterest.systems/android/switch) instead.
+    - When visually, it’s hard to tell that a checkbox turns something on or off. Use [Switch](https://gestalt.pinterest.systems/android/switch) instead.
+  </Group>
+  </TwoCol>
+
+## Best practices
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/32/33/bb/3233bb3a92c57dd29f2b5f2fd2a417f7.jpg" noPadding alt="Example of correct multi-select use" />
+    <Do title="Do" />
+    Use checkboxes for multi-selection of related list items
+  </Group>
+
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/1e/14/24/1e14245a26f12cf014028d24b00a6317.jpg" noPadding alt="Example of incorrect single-select use" />
+    <Dont title="Don't" />
+    Use checkboxes for one selection. Use [RadioGroup](https://gestalt.pinterest.systems/web/radiogroup) instead.
+  </Group>
+</TwoCol>
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/b2/20/93/b2209377241895d0471bde7340ee85c2.jpg" noPadding alt="Example of correct single checkbox use" />
+    <Do title="Do" />
+    Use a single Checkbox in forms where the selection only takes effect after submitting the form
+  </Group>
+
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/e7/07/02/e70702e40416c5f4f89e92eca2bbb490.jpg" noPadding alt="Example of incorrect immediate effect use" />
+    <Dont title="Don't" />
+    Use a Checkbox to turn a state on and off with immediate effect. Use [Switch](https://gestalt.pinterest.systems/android/switch) instead.
+  </Group>
+</TwoCol>
+
+## Accessibility
+
+People use Apple and Android’s accessibility features, such as VoiceOver and TalkBack to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple and Android’s accessibility documentation for more information:
+<TwoCol>
+  <Group>
+    [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/accessibility/overview/introduction/)
+    [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+  </Group>
+
+  <Group>
+    [Accessible design on Android](https://material.io/design/usability/accessibility.html#understanding-accessibility)
+    [Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility)
+  </Group>
+</TwoCol>
+
+## Writing
+
+<TwoCol>
+  <Group>
+    <Do title="Do" />
+    - Be clear and brief with checkbox labels so they are easily scanned
+  </Group>
+
+  <Group>
+    <Dont title="Don't" />
+    - Include lengthy text labels that make it hard for a user to scan a list of choices
+  </Group>
+</TwoCol>
+
+## Variants
+
+### State
+
+1. **Unchecked [Default]**
+1. **Checked**
+1. **Error**
+1. **Indeterminate**
+1. **Disabled**
+
+<ThreeCol>
```

Comment:
Is there a way to add space between these? they're kind of cramped 
<img width="930" alt="Screen Shot 2022-12-15 at 10 24 18 PM" src="https://user-images.githubusercontent.com/5125094/208014906-a6c35358-705c-4c16-a482-77925d9fdd99.png">